src/multipatch/connectivity/edge_transformation.hpp

```diff
@@ -84,43 +86,44 @@ class EdgeTransformation
      *      The current continuous dimension of the given coordinate coord. 
      * 
      * @return The analogous coordinate on the target patch. 
+     * @warning This operator is ill-defined when the two patches have the same
+     * continuous dimension. 
     */
     template <class CurrentDim>
     Coord<std::conditional_t<std::is_same_v<CurrentDim, EdgeDim1>, EdgeDim2, EdgeDim1>> operator()(
             Coord<CurrentDim> const& current_coord) const
     {
-        // The other continuous dimension
-        using TargetDim
-                = std::conditional_t<std::is_same_v<CurrentDim, EdgeDim1>, EdgeDim2, EdgeDim1>;
-
-        // The index range of CurrentDim
-        using IdxRangeCurrent = std::
-                conditional_t<std::is_same_v<CurrentDim, EdgeDim1>, IdxRangeEdge1, IdxRangeEdge2>;
-        // The index range of other dimension
-        using IdxRangeTarget = std::
-                conditional_t<std::is_same_v<CurrentDim, EdgeDim1>, IdxRangeEdge2, IdxRangeEdge1>;
-
-        // Get min and length on each 1D index ranges:
-        IdxRange<EdgeGrid1, EdgeGrid2> const
-                combined_idx_range(m_idx_range_patch_1, m_idx_range_patch_2);
-        IdxRangeCurrent const current_idx_range(combined_idx_range);
-        IdxRangeTarget const target_idx_range(combined_idx_range);
-
-        Coord<CurrentDim> const current_min = ddc::coordinate(current_idx_range.front());
-        double const current_length = ddcHelper::total_interval_length(current_idx_range);
-
-        Coord<TargetDim> const target_min = ddc::coordinate(target_idx_range.front());
-        double const target_length = ddcHelper::total_interval_length(target_idx_range);
-
-        double rescale_x = (current_coord - current_min) / current_length * target_length;
-
-        bool constexpr orientations_agree = Interface::orientations_agree;
-        if constexpr (!orientations_agree) {
-            rescale_x = target_length - rescale_x;
-        }
-        return target_min + rescale_x;
+        static_assert(
+                !std::is_same_v<EdgeDim1, EdgeDim2>,
+                "The two patches have the same conditinous dimension. "
+                "Please specify the input Patch as template parameter "
+                "using .template operator<CurrentPatch>()(current_coord).");
+        using CurrentPatch
+                = std::conditional_t<std::is_same_v<CurrentDim, EdgeDim1>, Patch1, Patch2>;
+        return (*this).get_equivalent_target_coordinate<CurrentPatch>(current_coord);
```

Comment:
```suggestion
        return get_equivalent_target_coordinate<CurrentPatch>(current_coord);
```
This is not needed to call operators with friendly names